src/server/services/hli/stadspas-gpass-service.ts

```diff
@@ -89,6 +92,19 @@ function transformStadspasResponse(
       0
     );
 
+    const passNumber = gpassStadspasResonseData.pasnummer;
+
+    let blockPassURL = null;
+    console.dir(gpassStadspasResonseData);
+    if (gpassStadspasResonseData.actief) {
+      const [encrypted] = encrypt(
+        gpassStadspasResonseData.pasnummer.toString()
+      );
+      blockPassURL = generateFullApiUrlBFF(BffEndpoints.STADSPAS_BLOCK_PASS, {
+        passNumber: encrypted,
+      });
+    }
```

Comment:
Soorgelijke functionaliteit hebben we al eens toegevoegd in `stadspas.ts:38-60`, kun je dat daar doen?